src/terra-dev-site/doc/how-to.2/change-application-theme.doc.mdx

```diff
@@ -0,0 +1,22 @@
+# How To Change the Application Theme
+
+The theme used by the application is set using the `themeName` prop on ApplicationBase.
+
+```diff
```

Comment:
Does this page need more discussion about how to set a default theme and IE10 Theme change limitations?

Reply:
Probably, but we're going to have to follow up on that later.